src/storage/mod.rs

```diff
@@ -14,6 +14,7 @@ use std::io::Read;
 use std::path::{Path, PathBuf};
 
 const MAX_CONCURRENT_UPLOADS: usize = 1000;
+const DEFAULT_COMPRESSION: CompressionAlgorithm = CompressionAlgorithm::Zstd;
```

Comment:
These could also be good targets for an environmental variable of some sort, we don't have multiple algos atm but in the future we might, and adjusting concurrent uploads would also be nice

Reply:
I don't think that's required for the current PR. Feel free to send a future PR though.

Reply:
With custom dictionaries the compression will probably be chosen on a per-file basis, so not easily configurable other than by changing code.